db/schema.rb

```diff
@@ -338,12 +351,18 @@
     t.string "uid"
     t.string "profile_picture_file_name"
     t.string "profile_picture_content_type"
-    t.bigint "profile_picture_file_size"
+    t.integer "profile_picture_file_size"
     t.datetime "profile_picture_updated_at"
     t.boolean "admin", default: false
     t.string "country"
     t.string "educational_institute"
-    t.boolean "subscribed", default: true
+    t.boolean "subscribed", default: false
+    t.boolean "accepted_privacy_policy", default: false
+    t.string "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string "unconfirmed_email"
+    t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true
```

Comment:
These fields are from unmreged PRs, right ?

Reply:
Yeah, I will reset this.